doc/build-windows.md

```diff
@@ -107,3 +132,20 @@ as they appear in the release `.zip` archive. This can be done in the following
 way. This will install to `c:\workspace\bitcoin`, for example:
 
     make install DESTDIR=/mnt/c/workspace/bitcoin
+
+Footnotes
```

Comment:
Maybe ot: For some reason GitHub doesn't insert links to the footnotes for me.

Reply:
Looks like that footnote is not supported on github. According to a comment on this SO [answer](https://stackoverflow.com/questions/25579868/how-to-add-footnotes-to-github-flavoured-markdown) that format is used on GitLab. I'll use the alternative suggestion of anchor links.

Reply:
This is great. I have tested on my WSL Ubuntu Xenial 16.04 several times and it works, but I have a question regarding this part: 
>sudo update-alternatives --config x86_64-w64-mingw32-g++

>There are 2 choices for the alternative x86_64-w64-mingw32-g++ (providing /usr/bin/x86_64-w64-mingw32-g++).

>  Selection    Path                                   Priority   Status
>------------------------------------------------------------
>  0            /usr/bin/x86_64-w64-mingw32-g++-win32   60        auto mode
>  1            /usr/bin/x86_64-w64-mingw32-g++-posix   30        manual mode
>  2            /usr/bin/x86_64-w64-mingw32-g++-win32   60        manual mode

The only choice is "1" to get bitcoin compiled successfully, the other choices cause errors in "make".  Is there a reason to leave 3 choices? Sorry, I'm still new to this and the choices made me confused at first until I tried them all.  
Thank you for doing this, I'm thrilled to be able to compile bitcoin on WSL again! :D

Reply:
Note that "0" is the default and the options are 1 (posix) and 2 (win32).